test/test_parse_psse.jl

```diff
@@ -37,6 +38,24 @@ end
     @test get_status(get_component(ThermalStandard, sys, "generator-2438-EG")) == 1
     @test get_available(get_component(ThermalStandard, sys, "generator-2438-EG")) == 1
 
+    sys3 = build_system(PSSEParsingTestSystems, "psse_ACTIVSg2000_sys")
+    sys4 = build_system(PSSEParsingTestSystems, "pti_frankenstein_70_sys")
+
+    base_dir = string(dirname(@__FILE__))
+    file_dir = joinpath(base_dir, "test_data", "5circuit_3w.raw")
+    sys5 = System(file_dir)
+
+    @info "Testing  Three-Winding Transformer Parsing"
+    @test isnothing(get_component(Transformer3W, sys3, "1"))
+    @test get_available(
+        get_component(Transformer3W, sys5, "FAV SPOT 01-FAV SPOT 02-FAV SPOT 03-i_A"),
+    ) == false
+    @test length(get_components(Transformer3W, sys5)) == 5
```

Comment:
Seems like there is more to test than this? Do we have coverage on all the three winding stuff that was added in this PR?